### PR TITLE
References added to docs

### PR DESCRIPTION
**Pull request by ceriottm**
Adding citation requirements to the main docs page

### CODE REVIEW

File: docs/src/index.rst [hunk 1]
```diff
@@ -20,6 +20,22 @@ different manners to create visualization adapted to different kinds of data. Th
...
+use it and incorporate it into your own research and software projects. 
+If you find it useful, we would appreciate a citation to the chemiscope 
+`paper`_
+
+G. Fraux, R. K. Cersonsky, M. Ceriotti, 
+Chemiscope: Interactive Structure-Property Explorer for Materials and Molecules. 
+Journal of Open Source Software 5 (51), 2117 (2020)
+
+If you incorporate chemiscope components into a software project, a link
+back to the `chemiscope`_ homepage is the preferred form of acknoqledgement.
```
Comment (Luthaf) [inline docs/src/index.rst:36]:
```suggestion
back to the `chemiscope`_ homepage is the preferred form of acknowledgement. 
```

**Comment by ceriottm** [comment]
Also make sure this addresses #63

**Comment by ceriottm** [comment]
> Looks good, should we also add this on the main page?

You mean that if we merge this it won't automatically update the main page?

**Comment by Luthaf** [comment]
This will update `https://chemiscope.org/docs/index.html`, I was asking about having something in `https://chemiscope.org/index.html`

**Comment by ceriottm** [comment]
We could include the JOSS markdown badge in the footer. I leave to you to decide if it looks good ^_^

**Comment by Luthaf** [comment]
Rebased & fixed conflicts

File: app/index.html [hunk 2]
```diff
@@ -173,13 +173,17 @@ <h5 class="modal-title">
                 <h5>References</h5>
                 <ul class="references">
                     <li>
-                        <a href="https://doi.org/10.1063/1.5090481">Atomic-Density representations for machine learning</a>
-                        <br/>MJ Willatt, F Musil, M Ceriotti - J. Chem. Phys, 2019
+                        <a href="https://doi.org/"10.21105/joss.02117>Chemiscope: interactive structure-property explorer for materials and molecule</a>
+                        <br/>G Fraux, RK Cersonsky, M Ceriotti - JOSS, 2020
                     </li>
                     <li>
                         <a href="https://doi.org/10.1039/c6cp00415f">Comparing molecules and solids across structural and alchemical space</a>
```
Comment (Luthaf) [inline app/index.html:180]: @ceriottm do we want to keep these references here now that we have dataset specific reference lists?

**Comment by ceriottm** [comment]
Probably not, I was thinking the same. Get them out.

**Pull request merged**